src/main.ts

```diff
@@ -427,11 +427,22 @@ ipcMain.handle(
 )
 
 // HAR
-ipcMain.handle('har:save', async (_, data: string, prefix?: string) => {
-  const fileName = generateFileNameWithTimestamp('har', prefix)
-  await writeFile(path.join(RECORDINGS_PATH, fileName), data)
-  return fileName
-})
+ipcMain.handle(
+  'har:save',
+  async (_, data: HarWithOptionalResponse, prefix: string) => {
+    const fileName = await generateUniqueFileName({
+      directory: RECORDINGS_PATH,
+      ext: '.har',
+      prefix,
+    })
+
+    await writeFile(
+      path.join(RECORDINGS_PATH, fileName),
+      JSON.stringify(data, null, 4)
```

Comment:
HARs and generators are now stringified just before writing to disk. I believe it makes more sense to do it this way (among other things it improves the DX since the param type is not `string` anymore).

Sidenote/question: why is it `4` for HARs and `2` for generators? 🤔

Reply:
I don’t have full context on the decision, but my guess is that 4 spaces were chosen for improved readability, especially since HAR files can become quite complex and deeply nested. 🤔

Reply:
Prettier defaults to `2`, so I'd say that's a good thing to go by.

Reply:
I don't think we were setting the padding so I'm not sure why they would be different 🤔